README.md

```diff
@@ -1,4 +1,4 @@
-# ![Intel Device Plugins for Kubernetes](.intel-logo.png) Device Plugins for Kubernetes
+# Intel Device Plugins for Kubernetes
```

Comment:
`Intel ®` ?

Reply:
I thought about it based on what we discussed but this is more consistent with other the READMEs we have

Reply:
I'm referring to other repositories brandings, like https://github.com/intel/intel-graphics-compiler

Reply:
`&reg;` added.